src/install/mongodb/intro.mdx

```diff
@@ -0,0 +1,22 @@
+---
+headingText: Before we start
+componentType: default
+---
+
+import infrastructureMongoDashboardCropped from 'images/infrastructure_screenshot-crop_mongodb-dashboard-cropped.webp'
+
+Our MongoDB integration collects and sends metrics from your MongoDB database to our platform, where you can monitor throughput, collection size, connections, indexes, and more
+
+With this data, you can use New Relic to create alerts when throughput spikes, create dashboards to monitor collection size, and more.
+
+<img
+  title="MongoDB quickstart dashboard"
+  alt="Image of the dashboard available through the MongoDB quickstart"
+  src={infrastructureMongoDashboardCropped}
+/>
+
+<figcaption>
+  Dashboard installed through the [New Relic MongoDB Monitor quickstart](https://newrelic.com/instant-observability/mongodb-prometheus).
+</figcaption>
```

Comment:
I'd consider removing the quickstart link. From a reader perspective, I want to click that link before even getting to the installation steps in this doc. It makes me not sure if I should follow the steps in the doc or go straight to the UI. If your goal is to first push for the guided install, I'd remove it out of the image caption and make it more prominent.